### PR TITLE
Add a greeting feature

### PR DESCRIPTION
**Pull request by johndbritton**
This is a work in progress, I'm going to add a greeting to this applicaiton

### CODE REVIEW

File: feature.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Greeter
+  def initialize(name)
+    @name = name
+  end
```
Comment (johndbritton) [inline feature.rb]: It's great that I can instantiate a greeter, but it still doesn't do anything.

**Comment by muan** [comment]
:cool: I think this is good to :ship: now!

**Comment by jonesfish** [comment]
I will buy 10 copies of this product ;-)

**Comment by jonesfish** [comment]
@muan : could you add me into the nccu-taipiei team?

**Pull request merged**